src/components/_global/BalRangeInput/BalRangeInput.vue

```diff
@@ -4,7 +4,7 @@ import 'vue-slider-component/theme/antd.css';
 import { computed } from 'vue';
 import VueSlider, { DefineComponent as TVueSlider } from 'vue-slider-component';
 
-import { theme } from '@/../tailwind.config';
+import tailwindConfig from 'tailwind.config.js';
```

Comment:
Note that we are explicitly importing `from 'tailwind.config.js'`which is a vite alias (see vite.config.ts) because we need to use ES modules but tailwind + tailwind intellisense require commonJS import style to work properly.